app/views/register/asset/shares/SharePortfolioNameView.scala.html

```diff
@@ -38,7 +38,9 @@
           field = form("value"),
           label = messages("sharePortfolioName.heading"),
           labelClass=Some("heading-large"),
-          labelAsHeading = true
+          labelAsHeading = true,
+          autocomplete = Some("name")
+
```

Comment:
@formHelper(action = ShareCompanyNameController.onSubmit(mode, index, draftId), 'autoComplete -> "off")
..... should 'autoComplete->"on" ?